manim/scene/scene_file_writer.py

```diff
@@ -12,8 +12,8 @@
 from ..constants import STREAMING_IP
 from ..constants import STREAMING_PORT
 from ..constants import STREAMING_PROTOCOL
-from ..constants import VIDEO_DIR
-from ..constants import VIDEO_OUTPUT_DIR
+from ..dirs import MEDIA_DIR
```

Comment:
The earlier problem was caused because directory paths were imported, why do it again?

Reply:
The earlier problem came mainly from the way they were important. Global for one, and also in a file that occasionally changes, forcing people to remove and re-add their own paths.

Also while piping towards scene_file_writer worked, I would've had to lay similar pipes to mobject, text and tex_mobject.
That would've made these pieces of code much more complicated for no real gain I see over this method.

Reply:
Got the point. The subsequent commits changed the imports.